feature-matcher-generator/src/main/java/ru/yandex/qatools/processors/matcher/gen/MatcherFactoryGenerator.java

```diff
@@ -62,7 +65,8 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
 
             VelocityEngine engine = engine();
 
-            FillMapWithFieldsProcess fillMapWithFields = fillMapOfClassDescriptionsProcess();
+            FillMapWithFieldsProcess fillMapWithFields =
```

Comment:
As you wish (JFYI: As I know, Oracle suggests to use only 80 symbols per line and Google suggests to use 80-100 symbols per line).